bundles/org.openhab.binding.irobot/src/main/resources/OH-INF/config/thing.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:irobot:thing">
+		<parameter name="address" type="text">
```

Comment:
Is this renamed from `ipaddress` to `address` intentionally? Can this be kept as `ipaddress` to maintain backward compatibility? Or is there more in this PR which is not backward compatible?

Reply:
As far as I know, it was part of a bigger refactoring. For now I changed it back to `ipaddress` to preserve backward compatibility.
(We can rename it to "address" together with the other refactoring and break the compatibility only once.)

Reply:
That's a good approach!